src/program/snabbnfv/neutron2snabb/neutron2snabb.lua

```diff
@@ -96,8 +96,13 @@ function filter (port, secbindings, secrules, direction)
          end
       end
    end
-   if #rules > 0 then return { rules = rules }
-                 else return nil end
+   if type == "stateless" then
```

Comment:
@n-nikolaev Does `--no-security-groups` work as expected today or do we need to implement that explicitly?